chapters/01-valores-tipos-operadores.md

```diff
@@ -81,19 +81,19 @@ Para subtração existe o operador `-` e para a divisão usamos o operador `/`.
 
 Quando os operadores aparecem juntos sem parênteses, a ordem que eles serão aplicados é determinada pela _precedência_ deles. O exemplo mostra que a multiplicação ocorre antes da adição. O operador `/` possui a mesma precedência que `*` e, de forma similar, os operadores `+` e `-` possuem a mesma precedência entre si. Quando vários operadores de mesma precedência aparecem próximos uns aos outros, como por exemplo `1 - 2 + 1`, eles são aplicados da esquerda para a direita: `(1 - 2) + 1`.
 
-Essas regras de precedência não são algo que você deve se preocupar. Quando estiver em dúvida, apenas adicione os parênteses.
+Essas regras de precedência não são coisas com as quais você deva se preocupar. Quando estiver em dúvida, apenas adicione os parênteses.
```

Comment:
Bacana, optei por "com as quais você deve" porque soa mais coesivo a natural. Obriado pela força!